### PR TITLE
Remove secondary-action-link component

### PR DESCRIPTION
**Pull request by lfdebrux**
We don't need to use a template or custom CSS to insert a link.

This PR also changes the copy of the link to be clearer; thanks @aliuk2012 for the suggestion.

### Before

image

### After

image

### CODE REVIEW

**Comment by aliuk2012** [comment]
## Great work! 😄👍

File: app/templates/briefs/application_submitted.html [hunk 1]
```diff
@@ -77,12 +77,11 @@ <h2 class="heading-xmedium">Evaluation</h2>
...
-  {%
-    with
-      url = url_for(".opportunities_dashboard", framework_slug=brief.frameworkSlug),
-      text = "Your {} opportunities".format(brief.frameworkName)
-  %}
-    {% include "toolkit/secondary-action-link.html" %}
-  {% endwith %}
+<a
+  class="govuk-link govuk-link--no-visited-state govuk-!-margin-top-6 govuk-!-display-inline-block"
+  href="{{ url_for(".opportunities_dashboard", framework_slug=brief.frameworkSlug) }}"
```
Comment (katstevens) [inline app/templates/briefs/application_submitted.html]: Single quote around `.opportunities_dashboard`?

File: app/templates/briefs/check_your_answers.html [hunk 2]
```diff
@@ -75,7 +75,7 @@ <h1 class="govuk-heading-l">
         class="govuk-link govuk-link--no-visited-state govuk-!-margin-top-6 govuk-!-display-inline-block"
         href="{{ url_for(".opportunities_dashboard", framework_slug=brief.frameworkSlug) }}"
```
Comment (katstevens) [inline app/templates/briefs/check_your_answers.html]: Single quoting needed here too

File: app/templates/briefs/application_submitted.html (hunk 1 above)
Comment (lfdebrux) [reply app/templates/briefs/application_submitted.html]: Thanks for catching that... again... I swear I'm not doing this on purpose 😅

File: tests/app/main/test_briefs.py [hunk 3]
```diff
@@ -1579,10 +1579,10 @@ def test_check_your_answers_page_shows_view_opportunity_link_for_submitted_appli
         view_your_opportunities_link = doc.xpath(
             '//a[@href="{0}"][contains(normalize-space(text()), normalize-space("{1}"))]'.format(
                 "/suppliers/opportunities/frameworks/digital-outcomes-and-specialists",
-                "Your Digital Outcomes and Specialists opportunities",
+                "Return to your opportunities",
             )
         )
-        assert len(view_your_opportunities_link) == 2  # Including breadcrumb link
+        assert len(view_your_opportunities_link) == 1
```
Comment (katstevens) [inline tests/app/main/test_briefs.py]: So we are no longer asserting breadcrumbs in this test? Probably... fine...?

**Comment by katstevens** [comment]
Not introduced by this PR, but I spotted a very small typo on the `application_submitted.html` breadcrumbs on the before/after images above: 

`"Responses to ‘{}’ submitted".format(brief.title)` 
should be
`"Response to ‘{}’ submitted".format(brief.title)`

The supplier can only respond once per brief! 😸

File: app/templates/briefs/application_submitted.html (hunk 1 above)
Comment (katstevens) [reply app/templates/briefs/application_submitted.html]: Surely we must be able to pick this up with a linter somehow?

Comment (lfdebrux) [reply app/templates/briefs/application_submitted.html]: I'll add a tech debt ticket for us to talk about Jinja/HTML linting

File: tests/app/main/test_briefs.py (hunk 3 above)
Comment (lfdebrux) [reply tests/app/main/test_briefs.py]: That was my thought process...

Comment (lfdebrux) [reply tests/app/main/test_briefs.py]: It looks like there is an existing test specifically for the breadcrumbs for this page https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/blob/master/tests/app/main/test_briefs.py#L2022-L2043

**Pull request merged**